src/components/form/templates/array-field-collapsible-item.ts

```diff
@@ -70,6 +70,12 @@ export class CollapsibleItemTemplate extends React.Component {
                 className: 'limel-form-array-item--object',
                 ref: 'section',
                 'is-open': this.isOpen,
+                'state-key': encodeURIComponent(
```

Comment:
Since we've changed the implementation from being built into `limel-collapsible-section`, to being built separately in `limel-collapsible-section-stateful`, this needs to be changed a bit, so that instead of just setting a property, we also switch which component we are using…

Thus the draft-PR…